samples/switchless/README.md

```diff
@@ -114,14 +132,21 @@ oeedger8r ../switchless.edl --untrusted
 
 ## About the host
 
-The host first defines a structure specifically for configuring switchless calls. In this case, we specify the
-first field `1` as the number of host worker threads for switchless OCALLs. In this example, 1) There is at most
-1 enclave thread all the time, and 2) The number of cores available to the host worker threads is unknown, and
-so we use 1 as explained above. The 2nd field specifies the number of enclave threads for switchless ECALLs.
-Since switchless ECALL is not yet implemented, we require the 2nd field to be `0`.
+The host first defines a structure specifically for configuring switchless calls.
+
+We conservatively assume that we are executing on a machine with 2 cores. The main thread will keep posting
```

Comment:
I think the first sentence should say something like:
"We conservatively assume that we are executing on a machine with 1 available core."  (i.e., 1 not 2, to be conservative)

Reply:
I agree. `Conservatively` does not sound right here. Will try to reword the paragraph minimally.

Reply:
I have rephrased it to..

"In this sample, the main thread will first keep posting switchless ocalls from within the enclave. To service these ocalls, .... One a machine with atleast two cores, both the caller and worker threads can potentially run uninterrupted on dedicated cores, causing less contention and leading to maximum speedup via switchless calling. On a machine with more than two cores, ..."